R/assertions.R

```diff
@@ -463,12 +463,12 @@ assert_filter_cond <- function(arg, optional = FALSE) {
 #'
 #' @param arg A function argument to be checked
 #'
-#' @param optional Is the checked parameter optional? If set to `FALSE` and `arg`
-#' is `NULL` then an error is thrown
-#'
 #' @param expect_names If the argument is set to `TRUE`, it is checked if all
 #'   variables are named, e.g., `vars(APERSDT = APxxSDT, APEREDT = APxxEDT)`.
 #'
+#' @param optional Is the checked parameter optional? If set to `FALSE` and `arg`
+#' is `NULL` then an error is thrown
+#'
 #' @author Samia Kabi
```

Comment:
Can you put your name as author Stefan as well since Samia is no longer involved.  Actually, been thinking more about authors - maybe this is something that is no longer needed??  I like single point of accountability...but...gets out of date quickly.

Reply:
Should be updated via https://github.com/pharmaverse/admiral/issues/1673.